### PR TITLE
Impl k8s openapi models

### PR DESCRIPTION
**Pull request by utaal**

### CODE REVIEW

File: src/resources/mod.rs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+use crate::pervasive::string::*;
+
+// use k8s_openapi::api::core::v1::ConfigMap as K8SConfigMap;
+use k8s_openapi::apimachinery::pkg::apis::meta::v1::ObjectMeta as K8SObjectMeta;
+
+verus! {
+
+#[verifier(external_body)]
+pub struct ObjectMeta {
+    inner: K8SObjectMeta,
```
Comment (lalithsuresh) [inline src/resources/mod.rs:13]: Are there restrictions on working with `inner` from the methods below (impl ObjectMeta....)?

Comment (utaal) [reply src/resources/mod.rs:13]: Yes, only trusted `#[verifier(external_body)]` functions can work with `inner`, which is why verified code will have to access it through trusted functions (like `name`).

**Comment by marshtompsxd** [comment]
I will work on top of this PR

**Comment by marshtompsxd** [comment]
Move to #74 and close this one

**Pull request closed without merging**